packages/dds/tree/src/test/feature-libraries/sequence-field/invert.test.ts

```diff
@@ -20,7 +20,7 @@ import { invert as invertChange, assertChangesetsEqual, tagChangeInline } from "
 import { ChangeMaker as Change, MarkMaker as Mark } from "./testEdits.js";
 
 function invert(change: SF.Changeset, tag: RevisionTag = tag1): SF.Changeset {
-	return invertChange(tagChangeInline(change, tag));
+	return invertChange(tagChangeInline(change, tag), tag);
```

Comment:
This is using the same tag for the input changeset and the output (inverse) changest.
It would be best to just declare a new tag in the module:
`const tagOut: RevisionTag = mintRevisionTag();`
And use it like so:
```suggestion
	return invertChange(tagChangeInline(change, tag), tagOut);
```